package.json

```diff
@@ -63,9 +62,12 @@
     "node": "0.10.29"
   },
   "scripts": {
+    "build": "grunt",
     "test": "grunt test",
```

Comment:
Seems like the grunt task for test is doing a little more that just running tests...